hack/makefile-vars.mk

```diff
@@ -0,0 +1,25 @@
+# Build options
+IMG_BUILD_CMD ?= docker
+TARGET_OS ?= linux
+TARGET_ARCH ?= amd64
+
+# Tools versions
+OPM_VERSION ?= v1.26.0
+
+# Tools paths
+OUTDIR ?= build/_output
+OUTDIR_TOOLS= ${OUTDIR}/tools
+OPM ?= ${OUTDIR_TOOLS}/opm-${OPM_VERSION}
```

Comment:
OPM vars in this file is shadowed by vars in script. Also, it's confusing to have same vars in multiple places.
Either move these to the script or move the script content to makefile.

Reply:
As said above, Can't be done in this case right now. I need the vars just here coz I need the path to the OPM tool.
The task of moving all the ensure tools to the new format is out of scope of this PR.
But I need the OPM path during the calling of the catalog build. This looks inconsistent right now, agreed. but have to keep it like this till I refactor the ensure files.